dabudimesh/shell.py

```diff
@@ -42,7 +48,8 @@ def do_message(self, arg):
         "message [addr] [msg] : Send message (msg) to node with address (addr)"
         args = self.__process(arg, 2)
         if args is not None:
-            self.router.send(args[0], args[1])
+            message = Message(self.router.address, "message", args[0], -1, args[1])
+            self.router.send(message)
```

Comment:
I think it is better to add send_text interface to router, which would be just with addr and text arguments